test/httpserver.ts

```diff
@@ -62,6 +72,13 @@ export class HttpServer {
     return nodeFetch.default(`${ensureAddress(this.hs)}${requestPath}`, opts)
   }
 
+  /**
+   * Open a websocket against the requset path.
```

Comment:
```suggestion
   * Open a websocket against the request path.
```

Reply:
[asking to learn/understand]: why do we open a new websocket against the request path?

Reply:
This is to test that the plugin was able to correctly register a websocket route at that path and respond.